fix: LEAP-447: Fix Number tag validation

robot-ci-heartex opened the pull request:
Hi @Gondragos!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7294303384) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1649

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7294303384

Gondragos committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7294303384

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7294397312

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7294835330

Commit:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7335742124

Commit:
Merge branch 'origin/develop' into 'fb-leap-447/number-validation'

Gondragos committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7336216580

Gondragos commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7718996555)
> Unresolvable merge conflict detected, please resolve it using the git command line: label_studio/frontend/dist/lsf/js/main.js
label_studio/frontend/dist/lsf/js/main.js.map
label_studio/frontend/dist/lsf/version.json

Gondragos commented:
/frontend build lsf
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7719786505)
> Successfully pushed new changes
>  heartexlabs/label-studio-frontend@524aa7db9517fb509cad0fa1146235577033b201

Gondragos committed:
[frontend] Get build lsf

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7719786505

Gondragos committed:
Merge branch 'develop' into fb-leap-447/number-validation

# Conflicts:
#	label_studio/frontend/dist/lsf/js/main.js
#	label_studio/frontend/dist/lsf/js/main.js.map
#	label_studio/frontend/dist/lsf/version.json
